spec/features/signing_in_as_different_types_of_user_spec.rb

```diff
@@ -44,6 +44,14 @@
     end
   end
 
+  scenario 'signing in should not be case-sensitive in matching the email address (bug 555)' do
```

Comment:
We have a feature spec for this already: https://github.com/DFE-Digital/get-help-with-tech/blob/main/spec/features/session_behaviour_spec.rb#L126